atlp-rwanda/cabal-phantom-backend
# #FMtr8tcx-setting-node.js-REST-API-empty-project-with-Documentation-by-Swagger

**What does this PR do?**
-showing documentation of welcome endpoint
 **Description of Task to be completed?**
-this PR it contain start server 
-Also has welcome endpoint and  its documentation by using swagger
**How should this be manually tested?**
1. type git checkout -b origin/FMtr8tcx_setting_SwaggerAsIPADoc 
2.type npm run dev to start server
3. Go to your browser and type in http://localhost:2301/api/docs/ to see documentation
**What are the relevant pivotal tracker stories?**
https://trello.com/c/FMtr8tcx/23-set-up-nodejs-rest-api-empty-project
**Screenshots (if appropriate)**
<img width="1440" alt="Screen Shot 2020-09-23 at 17 59 40" src="https://user-images.githubusercontent.com/41394162/94038355-9d24ad80-fdc6-11ea-80ac-199a32f59d39.png">
